### PR TITLE
feat(plugins): add support for avante.nvim

### PR DESCRIPTION
**Pull request by aarnphm**
Signed-off-by: Aaron Pham <contact@aarnphm.xyz>

### CODE REVIEW

File: lua/rose-pine.lua [hunk 1]
```diff
@@ -959,6 +959,14 @@ local function set_highlights()
 		GrugFarResultsMatch = { link = "CurSearch" },
 		GrugFarResultsPath = { fg = palette.foam },
 		GrugFarResultsStats = { fg = palette.iris },
+
+		-- yetone/avante.nvim
+		AvanteTitle = { fg = pallete.highlight_high, bg = pallete.rose },
+		AvanteReversedTitle = { fg = pallete.rose },
+		AvanteSubtitle = { fg = pallete.text, bg = pallete.foam },
```
Comment (aarnphm) [inline lua/rose-pine.lua:966]:
```suggestion
		AvanteSubtitle = { fg = pallete.highlight_med, bg = pallete.foam },
```

**Comment by mvllow** [comment]
Thank you!

**Comment by dav-cho** [comment]
I think `pallete` should be `palette`, no?

```
Error while calling lua chunk: .../.local/share/nvim/lazy/rose-pine/lua/rose-pine.lua:964: attempt to index global 'pallete' (a nil value)
```

**Comment by aarnphm** [comment]
oh no typo

**Pull request merged**